src/plugins/dashboard/kibana.json

```diff
@@ -20,7 +20,8 @@
     "uiActions",
     "urlForwarding",
     "presentationUtil",
-    "visualizations"
+    "visualizations",
+    "unifiedSearch"
```

Comment:
Need the `unifiedSearch` plugin so that I can access the `FilterItems` component for rendering the filter pills inside the modal rather than coming up with my own logic for this.